kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -359,7 +359,30 @@ public class FunSpec private constructor(
       typeVariables += typeVariable
     }
 
-    @JvmOverloads public fun receiver(
+    public fun contextReceiver(receiverTypes: Collection<TypeName>): Builder = apply {
```

Comment:
This is too many overloads IMO. I think we should stick with just two

```
addConextReceiver(TypeName)
addConextReceivers(Iterable<TypeName>)
```

Reply:
the receiver function has 3 overloads as well, I would prefer to have at least the vararg overloads

Reply:
I'm ok with vararg 👍